src/MongoDB.Driver.Core/Core/NativeLibraryLoader/macOS/NativeMethods.cs

```diff
@@ -0,0 +1,61 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using MongoDB.Driver.Core.Misc;
+
+namespace MongoDB.Driver.Core.NativeLibraryLoader.MacOS
+{
+    internal class NativeMethods : INativeLibraryLoader
```

Comment:
Not tested yet. Can be commented out. Or I can add a comment like it's not tested yet.